Client/Frontend/Browser/NightModeHelper.swift

```diff
@@ -37,17 +43,39 @@ class NightModeHelper: TabHelper {
         // Do nothing.
     }
 
+    static func setBrightness(_ value: CGFloat, animated: Bool) {
+        let screen = UIScreen.main
+        if animated {
+            brightnessQueue.cancelAllOperations()
```

Comment:
I know we only call `setBrightness` with animated=true but just to be safe I think you probably want to `cancelAllOperations` outside of the animated block.